string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,11 @@
+/* exported isVowel */
+function isVowel(char) {
+  var letter = ['a', 'E', 'I', 'o', 'u'];
```

Comment:
What about the other vowels? You only have some capitals and some lowercase letters here. It looks like you built this with specifically the test in mind.